app/checkout/[userId]/page.tsx

```diff
@@ -57,22 +57,17 @@ export default function CheckOut({ params }: CheckOutProps) {
         router.push(`/`);
     };
 
-    const onChange = (visitData: Visit) => {
-        setVisitFormData(visitData);
-    };
-
     const saveVisit = async (visitData: Visit) => {
         await updateVisit(visitData, params.userId);
         const client = await updateClient({
             ...clientData,
         });
```

Comment:
can we remove the unnecessary client update?

Reply:
Sure. Thanks for pointing out!